elasticstack/initialize.go

```diff
@@ -0,0 +1,69 @@
+package elasticstack
+
+import (
+	"strings"
+
+	"github.com/elastic/go-elasticsearch/v8"
+
+	_ "github.com/lib/pq"
+
+	"github.com/LetsFocus/goLF/configs"
+)
+
+type config struct {
+	addresses []string
+	user      string
+	password  string
+	retry     int
+}
+
+func cleanAddresses(addressesString string) []string {
+	addressesList := strings.Split(addressesString, ",")
+
+	var addressesSlice []string
+
+	for _, address := range addressesList {
+		trimmedAddress := strings.TrimSpace(address)
+		addressesSlice = append(addressesSlice, trimmedAddress)
+	}
+
+	return addressesSlice
+}
+
+func InitializeES(configs configs.Config, prefix string, retryCounter int) (*elasticsearch.Client, error) {
+	cfg := config{
+		addresses: cleanAddresses(configs.Get(prefix + "ELASTIC_ADDRESSES")),
```

Comment:
lets take  ELASTICSEARCH_HOST, ELASTICSEARCH_PORT seperately and form address using them
[]string{fmt.Sprintf("http://%s:%s", c.host, c.port)} use this and try